server/notification-providers/discord.js

```diff
@@ -33,84 +39,89 @@ class Discord extends NotificationProvider {
                 return okMsg;
             }
 
+            const embedFields = [
+                {
+                    name: "Service Name",
+                    value: monitorJSON.name,
+                },
+                ...(hasAddress ? [{
+                    name: "Service URL",
+                    value: address
+                }] : []),
+                {
+                    name: `Time (${heartbeatJSON.timezone})`,
+                    value: heartbeatJSON.localDateTime,
+                },
+                {
+                    name: "Error",
+                    value: msg,
```

Comment:
https://github.com/louislam/uptime-kuma/pull/5181#discussion_r1794703495

```suggestion
                    value: heartbeatJSON["msg"] ?? msg,
```